run-service.sh

```diff
@@ -18,8 +18,8 @@
 
 ARGS=$@
 APPHOME=home
-. apphome.sh
 
+MAVEN_OPTS="-agentlib:jdwp=transport=dt_socket,server=y,address=8000,suspend=n" \
```

Comment:
This looks creepy! Can you give some explanation here and in `run.sh`? Maybe add a comment of what this line does. Also, can you give a brief explanation of the difference between these two lines?